CHANGELOG.md

```diff
@@ -10,6 +10,11 @@
 - Added support for flag arguments in `CommandLineParser`. [#1409](https://github.com/spatialos/gdk-for-unity/pull/1409)
 - Added `Scripting Backend` option dropdown to the Build Configuration UI. [#1411](https://github.com/spatialos/gdk-for-unity/pull/1411)
 
+### Breaking Changes
+
+- All generated C# enums will now start from 0, being shifted to schema values on serialization and shifted back to C# values on deserialization. [#1412](https://github.com/spatialos/gdk-for-unity/pull/1412)
```

Comment:
Typically we put 'Breaking Changes' at the top as that will be most important to a developer when upgrading